apis/enigma-mocker/src/index.js

```diff
@@ -1,5 +1,50 @@
-import fromGenericObjects from './from-generic-objects';
+import fGO from './from-generic-objects';
 
-export default {
-  fromGenericObjects,
+/**
+ * @interface EnigmaMockerOptions
+ * @property {number} delay Simulate delay (in ms) for calls in enigma-mocker.
+ * @description Mocks Engima app functionality for demo and testing purposes.
```

Comment:
I suggest to change the description to this:

```suggestion
 * @description Options for Enigma Mocker
```

Maybe add the current description to the EnigmaMocker docs instead ?